pkg/views/workspace/selection/view.go

```diff
@@ -70,7 +81,24 @@ func (m model[T]) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				m.choice = &i.choiceProperty
 			}
 			return m, tea.Quit
+
+		case "d":
```

Comment:
I would prefer we use the `enter` key as confirmation to delete. It's much more intuitive.
Also, seems that pressing `enter` now just exists the command with no output.

Reply:
Fixed this. Now `enter` would work for multiple choices as well